classes/image_types_ostree.bbclass

```diff
@@ -145,16 +130,25 @@ IMAGE_CMD_ostree () {
 
     # Copy image manifest
     cat ${IMAGE_MANIFEST} | cut -d " " -f1,3 > usr/package.manifest
+}
 
-    cd ${WORKDIR}
+IMAGE_TYPEDEP_ostreetar = "ostreerootfs"
+IMAGE_CMD_ostreetar () {
+    ${IMAGE_CMD_TAR} -C ${OSTREE_ROOTFS} --numeric-owner --xattrs --xattrs-include='*' -cjf ${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}${OSTREE_TAR_SUFFIX} . || [ $? -eq 1 ]
+    ln -s ${IMAGE_NAME}${IMAGE_NAME_SUFFIX}${OSTREE_TAR_SUFFIX} ${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}${IMAGE_NAME_SUFFIX}${OSTREE_TAR_SUFFIX}
```

Comment:
One small more comment: Can we not rely on `create_symlinks`?

Reply:
Sorry, I think I probably mis-understood your comment, did you suggest we use subimages instead of "ln -s"?

Reply:
No I meant using the postfunc create_symlink. But yeah, it probably would require renaming... So maybe better leave as is, at least for now...